### PR TITLE
Tag XGBoost v0.2.0

### PR DESCRIPTION
**Pull request by slundberg**
A much needed tagging of a new release.

### CODE REVIEW

File: XGBoost/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.3-
```
Comment (tkelman) [inline XGBoost/versions/0.2.0/requires]: drop the trailing dash

Comment (tkelman) [inline XGBoost/versions/0.2.0/requires]: actually this doesn't support 0.3 at all as it uses `Ref`, so it should be updating the lower bound here - https://github.com/dmlc/XGBoost.jl/pull/29

**Comment by slundberg** [comment]
Thanks! We should have had 0.3 tests or explicitly changed support. Is there a simple alternative to `Ref` in 0.3? I would happy to keep support, but I think the c++ library interface changed which caused the use of `Ref` here.

Bumping the version dependency to `0.4` shouldn't be a big deal for this new version.

**Comment by slundberg** [comment]
I looked into making everything work on 0.3 but it ended up taking more time than I had, so I bumped the Julia version to 0.4.

**Comment by slundberg** [comment]
Thanks!

**Pull request merged**